CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 - Fixed a bug where the solid phase conductivity was double-corrected for tortuosity when loading parameters from a BPX file ([#2638](https://github.com/pybamm-team/PyBaMM/pull/2638)).
 - Changed termination from "success" to "final time" for algebraic solvers to match ODE/DAE solvers ([#2613](https://github.com/pybamm-team/PyBaMM/pull/2613)).
+- Changed depreciated installation link in CONTRIBUTING.md ([#2656](https://github.com/pybamm-team/PyBaMM/pull/2656)).
```

Comment:
I don't think this requires a changelog entry

Reply:
Okay, reverted the changelog entry. This PR might only have this URL fix as I have doubts regarding the rest.